lib/osf-components/addon/components/search-page/component.ts

```diff
@@ -156,4 +158,14 @@ export default class SearchPage extends Component<SearchArgs> {
         this.resourceType = resourceTypeOption.value;
         taskFor(this.search).perform();
     }
+
+    // analyticsEvent() logs category, actionName, label, and search query
+    @action
+    analyticsEvent(actionName: string) {
+        const q = this.searchText;
+        const category = this.resourceType;
+        if (q) {
+            this.analytics.track(category, actionName, 'search-page', q);
+        }
+    }
```

Comment:
I am not sure we want to actually track what users are searching.

Reply:
Following our meeting, this analytics service has been removed in favor of the more modern data analytics scope and name approach.